code/game/turfs/open.dm

```diff
@@ -228,6 +231,8 @@
 				wet_time = max(0, wet_time-10)
 			if(T0C + 100 to INFINITY)
 				wet_time = 0
+	else if (GetTemperature() > BODYTEMP_COLD_DAMAGE_LIMIT)	//seems like a good place
+		MakeDry(TURF_WET_PERMAFROST)
```

Comment:
It should frankly look cooler when atmos brings in a heater and the entire place starts melting